### PR TITLE
DT-4 update gitignore

### PR DESCRIPTION
**Pull request by andrejus-kuznecovas**
…tive, aspnetcore files

### CODE REVIEW

**Comment by andrejus-kuznecovas** [comment]
Test comment for slack+github

**Review by tomasmuzas — APPROVED** [review]
But do we really need that big .gitignore?

File: .gitignore [hunk 1]
```diff
@@ -259,4 +263,477 @@ paket-files/
...
\ No newline at end of file
+*.pyc
+
+# Cake - Uncomment if you are using it
+# tools/
+
+### ReactNative ###
+# React Native Stack Base
+
+.expo
```
Comment (NarijauskasMartynas) [inline .gitignore]: We are not using react-native libraries here

Comment (andrejus-kuznecovas) [reply .gitignore]: > 
> 
> We are not using react-native libraries here

will remove

Comment (NarijauskasMartynas) [reply .gitignore]: https://github.com/facebook/react/blob/master/.gitignore for react side, this one would be enough

**Pull request merged**